lib/screen/customer_support/support_thread_page.dart

```diff
@@ -605,12 +766,23 @@ class _SupportThreadPageState extends State<SupportThreadPage> {
     return result;
   }
 
+  int getRating(String text) {
+    try {
+      final prefix = text.substring(0, STAR_RATING.length);
+      final rating = int.parse(text.substring(STAR_RATING.length));
```

Comment:
using `int.tryParse` it could be more simple.

Reply:
you mean using tryParse to remove try catch, right. but we have to check text.length and check if rating is null

Reply:
it would be even more complicated, don't you think?

Reply:
I think you can check the text `startWith` first, then use `replacePrefix` to get the number. Then you won't need the try catch anymore

Reply:
Using try..catch will be slower than normal, so if something can be checked to bypass the try..catch, please do it first.